frontend/src/components/app-config/app-config-form.tsx

```diff
@@ -81,6 +83,28 @@ export const AppConfigForm: React.FC = () => {
             </FormItem>
           )}
         />
+        <FormField
+          control={form.control}
+          name="apptitle"
```

Comment:
can we change the field to just `title`?

Reply:
I found just "title" a bit ambiguous? But I am happy to change this if you prefer title. (Any alternative suggestions also welcome, of course.)

Reply:
sorry for the late response - maybe `app_title` then, instead of without a space

Reply:
after that, i think everything looks good! thanks for the awesome contribution

Reply:
> sorry for the late response - maybe `app_title` then, instead of without a space

Sounds good, I will change to that.